cpython/search_pypi_top.py

```diff
@@ -213,7 +215,9 @@ def search_dir(args, pypi_dir, pattern):
     projects = set()
     all_results = []
     regex = re.compile(pattern)
-    filenames = os.listdir(pypi_dir)
+    filenames = sorted(
+        (f for f in os.listdir(pypi_dir) if f not in IGNORED_FILENAMES), key=str.lower
+    )
```

Comment:
Can you please:

* Rename f to filename
* Put sorted() on a separated line: this one is too complicated

Reply:
Updated!